src/Entity/Conference.php

```diff
@@ -357,7 +357,7 @@ public function getCountry(): ?string
 
     public function setCountry(?string $country): self
     {
-        $this->country = $country;
+        $this->country = strtoupper($country);
```

Comment:
This will throw an error if `$country===null`, which is possible according to the signature.

And I would not to it here, but in the LocationGuesser Instead (and You already did it 👍🏼) so you can simply revert this :)